wd_core.au3

```diff
@@ -114,6 +114,7 @@ Global Enum _
 		$_WD_ERROR_FileIssue, _ ;
 		$_WD_ERROR_NotSupported, _ ;
 		$_WD_ERROR_AlreadyDefined, _ ; Used in _WD_CapabilitiesDefine and __WD_CapabilitiesInitialize
+		$_WD_ERROR_Javascript, _ ; Javascript error
```

Comment:
`$_WD_ERROR_JavaScript, _ ; JavaScript error`
instead
`$_WD_ERROR_Javascript, _ ; Javascript error`

?

Reply:
I'm using Camel Case naming convention, where the uppercase letter indicates the start of a new word. Even though JavaScript is traditionally written with a capitalized "S", I'm not doing it here because it isn't the start of a new word.